plugins/de.cau.cs.kieler.klighd.lsp/build.properties

```diff
@@ -0,0 +1,6 @@
+source.. = src/,\
+           xtend-gen/
+bin.includes = .,\
+               META-INF/,\
+               plugin.xml
+bin.excludes = **/*.xtend
```

Comment:
Are these build.properties correct?

Reply:
Ok, then add one.

Reply:
Only Eclipse Plug-Ins should have a plugin.xml, about.html and these as references in the build.properties file. The lsp and kgraph language server bundles are not meant to be used as Eclipse Plugins, therefore don't use these.